README.md

```diff
@@ -100,7 +100,7 @@ class Program
                 var dr = await p.ProduceAsync("test-topic", new Message<Null, string> { Value="test" });
                 Console.WriteLine($"Delivered '{dr.Value}' to '{dr.TopicPartitionOffset}'");
             }
-            catch (KafkaException e)
+            catch (ProduceException<Null, string> e)
```

Comment:
It's a pity this is more verbose. I'm tempted to just leave this as `KafkaException` since that's all that's needed by the log message, but it's instructional to use `ProduceException` since something this specific is always expected.

Reply:
I'm not thrilled with having the fully typed producer here.
Is there any way an untyped ProduceException can be used instead?

Reply:
there's always the base class `KafkaException`.